### PR TITLE
add default authors

### PR DESCRIPTION
**Pull request by mairaw**
Part III of ownership changes - update docfx to have default author for folders

Thoughts on whether I already include ms.author at the same time or not?

Next it would be removing individual authors from topics that have a central owner that is the same or that was updated.

### CODE REVIEW

File: docfx.json [hunk 1]
```diff
@@ -180,19 +180,82 @@
...
+        "docs/framework/winforms/**/**.md": "Thraka",
+        "docs/framework/wpf/**/**.md": "Thraka",
+        "docs/framework/xaml-services/**/**.md": "Thraka",
         "docs/fsharp/**/**.md": "cartermp",
         "docs/machine-learning/**/**.md": "natke",
-        "docs/visual-basic/**/**.md": "KathleenDollard",
-        "docs/spark/**/**.md": "mamccrea"
+        "docs/spark/**/**.md": "mamccrea",
+        "docs/standard/**/**.md": "gewarren",
+        "docs/standard/analyzers/**/**.md": "mairaw",
```
Comment (BillWagner) [inline docfx.json]: I thought this was something @gewarren was working on. (Not trying to change assignments, trying to proofread)

```suggestion
        "docs/standard/analyzers/**/**.md": "gewarren",
```

**Review by BillWagner — APPROVED** [review]
This LGTM @mairaw 

I had one comment for you and @gewarren to check. You can :shipit: when ready.

**Review by Thraka — APPROVED** [review]

File: docfx.json (hunk 1 above)
Comment (mairaw) [reply docfx.json]: We can change assignments if needed. I think we'll need to do one more round of assignments review with everyone now onboarded. But that matches what we discussed previously and https://github.com/dotnet/docs/blob/master/.github/CODEOWNERS#L111.

**Comment by mairaw** [comment]
I'll merge this one and get a second one for ms.author.

**Pull request merged**